docs/Advanced/SQL Searches/SQL Columns.md

```diff
@@ -0,0 +1,56 @@
+---
+publish: true
+---
+
+# SQL Columns
+
+<span class="related-pages">#advanced/sql-search</span>
+
+The following columns or properties are available to be used in the WHERE clauses when writing the SQL queries. The SQL engine also allows you to access the columns or properties children. For the remainder of this document the term column will be used to reference a property of the task that may or may not have sub properties you can access
+
+## Available Columns
+
+| Column Name              | Description                                                                         | Type                |
+| ------------------------ | ----------------------------------------------------------------------------------- | ------------------- |
+| status                   | The status object holds the current and next indicator and a description.           | Status Object       |
+| status->symbol           | The value between the square brackets. ('x', ' ', '-', '/', etc.)                   | string              |
+| status->name             | The display name for the status. ('Done', 'Todo', 'Cancelled', 'In Progress')       | string              |
+| status->nextStatusSymbol | The next indicator to be used when clicked on.  ('x', ' ', '/', etc.)               | string              |
+| description              | The description of the task.                                                        | string              |
+| indentation              | The indentation type used like \*, -, > -. etc.                                     | string              |
+| listMarker               | The list marker type, -, \* or NUMBER.                                              | string              |
+| taskLocation             | Location object holds the below properties                                          | TaskLocation Object |
+| taskLocation->lineNumber | Line number the task is on in the note.                                             | string              |
```

Comment:
when i first read the docs the "hold the below properties" statement was confusing because it seemed to imply al;l the following were properties of the object when only those with the -> were.  Not ice that for status object it doesn't say this same thing but has a description of the use/intent of the object.

Reply:
Sorry for not being clear.  Saying this implies nthere are several items that are part of the object and in this or perhaps it was another object there was just one, but the item after the obj-> item had a name that *might* have been a part of the object.  The use of "below" implies more than one time is possible and when a name *might* be part of an object it left me wondering if there was a docs error leaving out the "obj->" gfor that item.

Reply:
Sorry @pcause I still don't understand. You're replying to my comment about line number. I cannot see what code you are referring to and how it relates to the line number field.

Reply:
it was minor point.

Reply:
Updated documentation and removed reference.